server/config/filesystems.php

```diff
@@ -48,6 +48,11 @@
             'root' => storage_path('app'),
         ],
 
+        'backup' => [
+            'driver' => 'local',
+            'root' => storage_path('app/backups'),
+        ],
```

Comment:
We probably want to send the backup off-site through the internet (S3 free tier?). An improement for next task.